unit_converter/__init__.py

```diff
@@ -334,7 +328,7 @@ class Plugin(albert.TriggerQueryHandler):
             "C": "degC",
         },
         # Display names for units
-        # Units may be added here to override the default display names
+        # Units may be added here to o>verride the default display names
```

Comment:
probably added accidentially

Reply:
You're not wrong.

Fixed in 999c115e34e8d48cba2dec5e83596331837ec1eb